src/PrestaShopBundle/Controller/Admin/ModuleController.php

```diff
@@ -58,15 +58,18 @@ public function refreshCatalogAction(Request $request)
 
         $filters = new AddonListFilter();
         $filters->setType(AddonListFilterType::MODULE | AddonListFilterType::SERVICE)
-            ->setStatus(~AddonListFilterStatus::INSTALLED);
+            ->setStatus(~AddonListFilterStatus::INSTALLED)
+        ;
 
         try {
-            $products = $modulesProvider->generateAddonsUrls(
+            $modules = $modulesProvider->generateAddonsUrls(
```

Comment:
all are modules or addons, products is too disturbing as we manage products too.